moveit_core/robot_trajectory/test/test_robot_trajectory.cpp

```diff
@@ -634,6 +634,31 @@ TEST_F(OneRobot, UnwindFromState)
   }
 }
 
+TEST_F(OneRobot, MultiDofTrajectoryToJointStates)
```

Comment:
Nit: Do you mind adding a test description here, something like GIVEN, WHEN, THEN